zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collections.java

```diff
@@ -82,6 +83,7 @@
     private Function<ApproveTask, Future<Boolean>> addTaskToQueue = (task) -> ApprovalQueue.add(task);
     private BiConsumer<Collection, EventType> publishingNotificationConsumer = (c, e) -> new PublishNotification(c).sendNotification(e);
     private Function<Path, ContentReader> contentReaderFactory = (p) -> new FileSystemContentReader(p);
+    private Supplier<CollectionHistoryDao> collectionHistoryDaoSupplier = () -> getCollectionHistoryDao();
```

Comment:
Instead of creating a lambda function call the method, reference the function. Eg
```
    private Function<ApproveTask, Future<Boolean>> addTaskToQueue = ApprovalQueue::add;
    private BiConsumer<Collection, EventType> publishingNotificationConsumer = (c, e) -> new PublishNotification(c).sendNotification(e);
    private Function<Path, ContentReader> contentReaderFactory = FileSystemContentReader::new;
    private Supplier<CollectionHistoryDao> collectionHistoryDaoSupplier = CollectionHistoryDaoFactory::getCollectionHistoryDao;
```